pandas/core/array_algos/masked_reductions.py

```diff
@@ -119,11 +119,11 @@ def _minmax(
             # min/max with empty array raise in numpy, pandas returns NA
             return libmissing.NA
         else:
-            return func(values)
+            return func(values, axis=axis)
     else:
         subset = values[~mask]
         if subset.size:
-            return func(subset)
+            return func(subset, axis=axis)
```

Comment:
Note that now you are using `subset` again on this line, passing this `axis` is not doing anything (and would actually raise an error if you would pass `axis=1` here)

(it doesn't really matter in practice because we never call this with an axis=1, but seeing `axis` passed through might give the false impression that this algo actually supports 2D data, while that is not the case)

Reply:
@topper-123 can you address this?

Reply:
Oh, I thought I had answered this, apparently not...

`func` here is either `np.min` or `np.max`, so supplying `axis=axis` will not raise here, but will work as expected AFAIKS.

Additionally, without the `axis=axis` part, `func(subset)` is similar to `np.max|min(subset, axis=None)`. Not a problem for 1d arrays, but will be a problem if we ever want to support `df.min(axis=None)` using 2d masked arrays. (I'm not sure we want to support 2d masked arrays or are going all in on arrow?)